### PR TITLE
Motions and Disputes Reveal Vote Widget

### PR DESCRIPTION
**Pull request by rdig**
## Description

This PR adds in the voting reveal widget to the motion page, but does not wire in the saga

This widget has two states:
1. vote was not revealed yet, just show a info message
2. vote was revealed, show the user's vote

**Notes on testing**
To test this properly you'll need to 
- get two users in the colony
- make sure both have 50% reputation (eg: make a payment of 1000 to each user)
- create the motion
- stake both sides (you can do this with just one user)
- vote (one user for yes, another one for no)
- _at this point you can see the initial state of the reveal widget_
- reveal one user's vote
- _now you can see the vote, of the user who revealed_
- _log in with the user who didn't reveal to make sure that it doesn't show a revealed vote_

**Changes** 
- [x] Extract `VoteDetails` into separate component
- [x] Added `RevealWidget` Component
- [x] Added `motionUserVoteRevealed` client resolver
- [x] Added `MotionVote` enum type
- [x] Fixed `getMotionState` util

**Demo**

Both states, voting not revealed yet (first motion), vote revealed (second motion)

demo-md-reveal-widget-1

Vote not revealed user logged out / logged in

demo-md-reveal-widget-2

Vote revealed user logged out / logged in

demo-md-reveal-widget-3

User not logged in:
Screenshot from 2021-04-21 13-24-12

User logged in, who VOTED
Screenshot from 2021-04-21 13-24-17

User logged in, didn't VOTE
Screenshot from 2021-04-21 13-24-24

Resolves DEV-288

### CODE REVIEW

**Review by alicjakujawa — CHANGES_REQUESTED** [review]
Add handling the state, when user join to colony when motion was already in reveal state. (disable or hide reveal widget)

File: src/utils/colonyMotions.ts [hunk 1]
```diff
@@ -11,6 +11,11 @@ export enum MotionState {
   Invalid = 'Invalid',
 }
 
+export enum MotionVote {
+  Yay = 1,
+  Nay = 2,
```
Comment (rdig) [inline src/utils/colonyMotions.ts:16]: Lol! Nay is 0, not 2!

Please remember to fix this!

**Review by alicjakujawa — APPROVED** [review]
Just reminder:
* star alignment
* disable vote button

**Comment by rdig** [comment]
@alicjakujawa 

1. Disabled voting if the user doesn't have enough reputation

Screenshot from 2021-04-21 16-52-12

2. Properly aligned the reputation display icon (star)

Screenshot from 2021-04-21 16-49-22

**Pull request merged**